### ISSUE TITLE
Text for "lower details idling" does not appear at all, as if option doesn't exist

### ISSUE THREAD

**Issue by MRSS02** [LeonardSSH/vscord#33]
![image](https://user-images.githubusercontent.com/68190118/137059064-09c225eb-4d6d-48ec-b7e8-ef09f4ae8e74.png)

**Comment by MRSS02** [comment]
the text that should appear while hovering on the picture for idling (the keyboard) also does not show up at all.

**Comment by LeonardSSH** [comment]
More details, some reproduce settings?

**Comment by MRSS02** [comment]
![image](https://user-images.githubusercontent.com/68190118/147952133-ae9a36cf-e7ad-47e3-a600-af349b33aeac.png)

**Comment by LeonardSSH** [comment]
Oh, ok... I understand the problem better now. 

It is being solved soon™

**Comment by LeonardSSH** [comment]
The problem has been solved. I'll release a patch version very soon.

**Comment by LeonardSSH** [comment]
@MRSS02 

Resolving this issue, triggered another issue. Well, not really an issue, in the sense that, in RPC, Idling appeared twice, both in details and in states.
```jsonc
{
   // Must be set to false if you want the custom string to appear when idling.
   "rpc.removeLowerDetailsIdling": false
}
```

Resolved in a6e2c354bfb6c56440fe1f959207a5edd422d2ae.
